chpl/chpl-service/src/main/java/gov/healthit/chpl/builder/ComplaintsWorksheetBuilder.java

```diff
@@ -466,13 +469,13 @@ private void addComplaintData(final SurveillanceReportWorkbookWrapper workbook,
         addDataCell(workbook, row, COL_ONC_COMPLAINT_ID, complaint.getOncComplaintId());
         addDataCell(workbook, row, COL_SUMMARY, complaint.getSummary());
         addDataCell(workbook, row, COL_ACTIONS_RESPONSE, complaint.getActions());
-        addDataCell(workbook, row, COL_COMPLAINANT_TYPE, 
+        addDataCell(workbook, row, COL_COMPLAINANT_TYPE,
                 complaint.getComplainantType() != null ? complaint.getComplainantType().getName() : "");
         addDataCell(workbook, row, COL_COMPLAINANT_TYPE_OTHER, complaint.getComplainantTypeOther());
         addDataCell(workbook, row, COL_COMPLAINANT_CONTACTED, complaint.isComplainantContacted() ? "Yes" : "No");
         addDataCell(workbook, row, COL_DEVELOPER_CONTACTED, complaint.isDeveloperContacted() ? "Yes" : "No");
         addDataCell(workbook, row, COL_ATL_CONTACTED, complaint.isOncAtlContacted() ? "Yes" : "No");
-        addDataCell(workbook, row, COL_COMPLAINT_STATUS, complaint.getComplaintStatusType().getName());
+        addDataCell(workbook, row, COL_COMPLAINT_STATUS, complaint.getClosedDate() == null ? "Open" : "Closed");
```

Comment:
make the "Open" and "Closed" strings constants or pull from an enum or something